src/Settings/Path.hs

```diff
@@ -146,6 +150,12 @@ packageDbDirectory :: Stage -> FilePath
 packageDbDirectory Stage0 = buildRootPath -/- "stage0/bootstrapping.conf"
 packageDbDirectory _      = "inplace/lib/package.conf.d"
 
+-- | ( During installtion) Path to package database directory of a given 'Stage'.
```

Comment:
Perhaps, paths that refer to installation should live in a different file?

Can we keep all installation-related code in one file?